migrations/4_add_date_tracking_published_projects.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
if this is data that must not be lost, you migrate in two steps:
1. in the first migration, add the columns as _colname, with a down migration that does nothing, and then
2. in the second migration, rename the columns to their `colname` rather than `_colname`, with a down migration that simply reverses that rename.